isAOCTest.conf

```diff
@@ -1,3 +1,7 @@
+[indiserver]
+f = /opt/MagAOX/drivers/fifos/indiserver.ctrl
+
 [local]
-drivers=pdu0,pdu1,pdu2,pdu3,tcsi,adctrack,ktrack,labCool,sysMonAOC
+drivers=
+#drivers=pdu0,pdu1,pdu2,pdu3,tcsi,adctrack,ktrack,labCool,sysMonAOC
```

Comment:
The local.drivers configuration value has been left empty; resurrector gets this information from proclist_*.txt and passes it to the INDI server via the indiserver.ctrl FIFO.

This change has been made to every is*.conf in this directory.